src/stream/src/executor/hash_agg.rs

```diff
@@ -283,6 +295,7 @@ impl<K: HashKey, S: StateStore> HashAggExecutor<K, S> {
                     None
                 } else {
                     lookup_miss_count.fetch_add(1, Ordering::Relaxed);
+                    no_cache_miss_whole_chunk = false;
```

Comment:
What about just:

```rust
let no_cache_miss = futs.is_empty();
```

I think `no_cache_miss` is enough for this variable, since we are in `apply_chunk` and only process one chunk for one time.

Or just:

```rust
if !futs.is_empty() {
    chunk_lookup_miss_count.fetch_add(1, Ordering::Relaxed);
}
```

Reply:
Thanks! Let me try it, I thought 'None' also goes into the futs Vec before